app/app.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express');
 const createError = require('http-errors');
-const path = require('path');
 const cookieParser = require('cookie-parser');
 const logger = require('morgan');
 const passport = require('passport');
-
+const cors = require('cors');
```

Comment:
In the same vein, is it worth pursuing any form of rate limiting within the server application? Or should this be moved higher up in the stack?

Reply:
Probably just hide behind a Cloudflare Proxy Server.